master.tf

```diff
@@ -23,29 +23,37 @@ resource "linode_instance" "k8s_master" {
     kernel = "linode/direct-disk"
 
     devices {
-      sda = {
+      sda {
         disk_label = "boot"
       }
     }
   }
 
   provisioner "file" {
-    source      = "${path.module}/scripts/"
+    source      = "${path.cwd}/${path.module}/scripts/"
     destination = "/tmp"
 
     connection {
-      user    = "core"
-      timeout = "300s"
+      host        = self.ip_address
```

Comment:
Is the user's local IP address? Seems unnecessary.

Reply:
Negative -- the host line specifies the IP of the resource where the provisioner runs -- eg, the k8s master in this case. TF 0.12 no longer automatically provides any default for connection blocks, so specifying this is required.